doc/backend.md

```diff
@@ -9,12 +9,33 @@ In the documentation, TensorFlow {{ tensorflow_icon }} and PyTorch {{ pytorch_ic
 
 ### TensorFlow {{ tensorflow_icon }}
 
-TensorFlow 2.2 or above is required.
+- Model filename extension: `.pb`
+- Checkpoint filename extension: `.meta`, `.index`, `.data-00000-of-00001`
+
+[TensorFlow](https://tensorflow.org) 2.2 or above is required.
 DeePMD-kit does not use the TensorFlow v2 API but uses the TensorFlow v1 API (`tf.compat.v1`) in the graph mode.
 
 ### PyTorch {{ pytorch_icon }}
 
-PyTorch 2.0 or above is required.
+- Model filename extension: `.pth`
+- Checkpoint filename extension: `.pt`
```

Comment:
i might be wrong, I am testing the freeze and convert, below is what i see:

- for a TF checkpoint, I can simply do `dp freeze -o model.pb` and then `dp convert-backend model.pb model.pth`
- for a PT checkpoint, it seems I can directly convert the checkpoint `dp convert-backend model.ckpt.pt model.pb`, without doing freeze `dp --pt freeze -o model.pth`